docs/config.rst

```diff
@@ -403,6 +403,20 @@ brackets. For example::
 To learn how to publish your pipeline on GitHub, BitBucket or GitLab code repositories read :ref:`sharing-page`
 documentation page.
 
+nextflowVersion
+^^^^^^^^^^^
+
```

Comment:
Very good! RST format requires the `^^^` to have exactly the same length as the title.

Reply:
ok, did not notice this error when I build the documentation on my computer.
I'll update it.
Thanks

Reply:
Also, please use `Nextflow version` as title instead of the camel-case

Reply:
oauch .. too late 😬

Reply:
Np, I can make another commit if you want

Reply:
let's do!